services/galley/src/Galley/API/MLS/Message.hs

```diff
@@ -281,6 +286,7 @@ postMLSCommitBundleToLocalConv qusr conn bundle lcnv = do
                 /= Set.fromList (map (snd . snd) (cmAssocs (paAdd action)))
             )
             $ throwS @'MLSWelcomeMismatch
+        storeGroupInfoBundle lconv (cbGroupInfoBundle bundle)
```

Comment:
Isn't this too eagerly storing group info? Didn't we say this cannot be moved here as there are checks that have to run before this is stored?

Reply:
Thanks for catching it, I had forgotten. Fixed now.